css/CommonTheme.css

```diff
@@ -1,21 +1,24 @@
 /*------- Header Theme ---------*/
 
-.headerT{
+.navigation-bar-theme{
 	box-shadow: 0px 2px 2px #888888;
 	font-weight: 400;
 	font-size: 24px;
 }
 
-.header-elements-linkT{
+.navigation-bar-element-link-theme{
 	color: #414141;
 	text-align: center;
 	text-decoration: none;
 }
 
+.navigation-bar-active-page-theme{
+	border-color: #03a8f3;
+}
 
 /*-----------------   Footer -----------------*/
 
-.footerT{
+.page-footer-theme{
 	color:#828384;
 	background-color: #dfdfdf;
 	text-align: center; 
```

Comment:
I think you are trying to set the way for a later themeing support here, which is nice, but with that in mind would text-align and text-decoration belong here or not?

I also appreciate that you are setting up a naming convention, but you also use separators for the same purpose for example `navigation-bar-active-page` might make a lot more sense like this `navigationBar-activePage` wouldn't you say?

Reply:
Yes it seems an issue the naming the way I do as soon as I hit more than 3 words. Wanted to firmly respect a naming convention that is why stood so much with the '-' and didn't want to include camel cases. 
Will try to think of a way to remake the naming for the more complex CSS structures